docs/spec/sanitations.md

```diff
@@ -1,24 +1,50 @@
-_Author_:  <!-- TODO: Add author name --> \
-_Created_: <!-- TODO: Add date --> \
+_Author_:  Senuri Mallikarachchi\
+_Created_: 2025/01/03\
 _Updated_: <!-- TODO: Add date --> \
 _Edition_: Swan Lake
 
 # Sanitation for OpenAPI specification
 
 This document records the sanitation done on top of the official OpenAPI specification from HubSpot Marketing Subscriptions. 
-The OpenAPI specification is obtained from (TODO: Add source link).
+The OpenAPI specification is obtained from [Hubspot API Reference](https://github.com/HubSpot/HubSpot-public-api-spec-collection/blob/13f888343a8ec443d95de2c12393015ba3391ac6/PublicApiSpecs/Communication%20Preferences/Subscriptions/Rollouts/176901/v4/subscriptions.json).
 These changes are done in order to improve the overall usability, and as workarounds for some known language limitations.
 
 [//]: # (TODO: Add sanitation details)
-1. 
-2. 
-3. 
+1. Change the url property of the servers object:
+
+    - Original: ` https://api.hubspot.com `
+    - Updated: ` https://api.hubapi.com/communication-preferences/v4 `
+    - This change is made to ensure that all API paths are relative to the versioned base URL (/v4), which improves the consistency and usability of the APIs.
+
+2. Use `--nullable` option when generating the client using the Ballerina OpenAPI tool:
+
+    - The Hubspot API reference does not properly include the "nullable" property for some the request and response schemas.
+    - Therefore, the `--nullable` option is used as a precaution to avoid potential data-binding issues in the runtime, which will generate all the request/response type fields with the support to handle null values.
```

Comment:
```suggestion
    - Therefore, the `--nullable` option is used as a precaution to avoid potential data-binding issues in the runtime, which generates all the request/response type fields with the support to handle null values.
```